source/index.js

```diff
@@ -172,11 +172,13 @@ module.exports = async (input = 'patch', options) => {
 		task: () => git.push()
 	});
 
-	tasks.add({
-		title: 'Creating release draft on GitHub',
-		enabled: () => isOnGitHub === true,
-		task: () => releaseTaskHelper(options)
-	});
+	if (options.releaseDraft) {
```

Comment:
Are there any advantages to using this over the task's `skip` property?

Reply:
The task doesn't show up at all, using `skip` it would be shown as "skipped". I was thinking since this option would most likely be used in scripts that do the release through the API after using `np`, end users of those scripts might be confused by seeing "Skipped GitHub release draft".

Reply:
We could use `enabled`, which would hide the task from the user in case the flag is provided, though I think it's better to inform the user that the task is usually executed, but is skipped in this case because the `--no-release-draft` flag was provided.

Also, do you have an example for a script which does the release using `np`'s API? Usually I see people either manually running `$ np` or setting it to be the `release` npm script and running `$ npm run-script release`. In the two usecases I mentioned above, using `skip` shouldn't be confusing. It would be as _confusing_ as any other skipped task.

Reply:
@itaisteinherz unfortunately the repo is private so I cannot share the script. The specific use case is for packages distributed via GitHub releases instead of NPM. The script calls `np` and the uses the GitHub API to create the working release.

It's ok for me to mark the task as skipped though, i'll change it